src/harvester/__tests__/queries.ts

```diff
@@ -17,3 +17,18 @@ describe('didsWithClaims', () => {
     expect(result.variables).toEqual(variables);
   });
 });
+
+describe('eventByIndexedArgs', () => {
+  test('should verify that the variables are the same that were passed to the query', () => {
```

Comment:
"should pass the variables to the grapqhl query"

(change the other test to say the same)